source/manual/set-up-aws-account.html.md

```diff
@@ -29,10 +29,28 @@ To sign in, go to [the GDS AWS Sign page](https://gds-users.signin.aws.amazon.co
 
 👉 [Sign in to AWS GDS account](https://gds-users.signin.aws.amazon.com/console)
 
-## 3. Set up your MFA
+## 3. <a id="set_up_mfa"></a>Set up your MFA
```

Comment:
Headings actually already get anchors, this one is `3-set-up-your-mfa`.

Reply:
Well that's awesome. Will change!